lib/y2_installbase.pm

```diff
@@ -628,6 +628,15 @@ sub save_remote_upload_y2logs {
     $self->investigate_yast2_failure();
 }
 
+# Builds RC1, PublicRC and GMC are public and can be identified by the following function during installation
+sub is_public_build {
```

Comment:
let's see what RMs reply (in slack) but I think the [phases](https://confluence.suse.com/display/SUSELinuxEnterpriseServer15SP6/Schedule) (check here at the bottom) are not connected with public/non-public, but idk. Perhaps we could call it `is_rc_gm_phase`.
(lower letter for method and variable names in my own experience helps with silly mistakes as the code doesn't compile and the IDE is not very helpful sometimes, but that is just my taste).

Reply:
another thought is that public beta is too soon to file this kind of bugs, from the experience from previous SP, some things like this are only tackled by developers from RC, another name, `is_dev_phase_rc_and_onwards`, but I prefer the other name above, definitely public might not be understood at all for the future user of this method.

Reply:
how about:
is_build_gt_beta to borrow the bash naming "greater than" or 
is_build_lt_beta "later than" 
or is_build_past_beta
I prefer the last one , with some clarifying comments in the lib

Reply:
pushed the last change so you can have an overview